examples/quarkus/testsuite/native/pom.xml

```
@@ -38,6 +38,15 @@
                 <module>mssql</module>
             </modules>
         </profile>
+        <profile>
+            <id>release</id>
```

The profile is called `blazebit-release`
```suggestion
            <id>blazebit-release</id>
```